### ISSUE TITLE
Alt text

### ISSUE THREAD

**Issue by farishkash** [armaninthegame/prj-rev-bwfs-tea-cozy#3]
Desscriptive alt text should be used for the images.  This helps visually impaired people know what the pictures are as screen readers will read off the alt text.
